index-variables.scss

```diff
@@ -0,0 +1,32 @@
+// utils
+$include-normalize-css: false;
```

Comment:
Instead of such a index-variables.scss I would like to have one variable $include-css which is the default for all variables.

Reply:
the normalize file didnt have any gateway anyways, so that wouldnt have helped, but i get your point.  fwiw (afaict) we will still need a separate index file to set the initial scss $include-css variable